SIS_import/sis_set_url.rb

```diff
@@ -38,6 +56,7 @@ def sleep_according_to_timer_and_api_call_limit(start_time, end_time, call_count
 		start_time = Time.now
 		end_time = start_time + time_interval_in_seconds # one minute apart
 		#rest the esb call count
+		p "reset call count"
```

Comment:
the call count is logged/printed just before this line. So there is no need to print again.